conda-store-server/conda_store_server/orm.py

```diff
@@ -54,39 +54,36 @@ class Namespace(Base):
 
     metadata_ = Column(JSON, default=dict, nullable=True)
 
-    role_mappings = relationship("NamespaceRoleMapping", back_populates="namespace")
```

Comment:
Not sure it's possible to have this field since I now use two foreign keys in the NamespaceRoleMapping table.
In any case, it doesn't seem to be necessary.
So I just removed it.